cylc/flow/scripts/cat_log.py

```diff
@@ -44,6 +44,9 @@
   # Print workflow log:
   $ cylc cat-log foo
 
+  # Print specific workflow log:
+  $ cylc cat-log foo -f 02-start-01.log
```

Comment:
To assist with displaying other Cylc log files e.g. `install/`, `conf/`, and wherever the version-control log goes, we might want to add the `log/` directory prefix e.g:

```
-f log/02-start-01.log
```